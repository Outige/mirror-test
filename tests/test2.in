5 10 10
scamp
oiler
wring
texan
seems
licks
aural
salsa
flaps
power
(sowtafpjkzme)(cireualoz)(alixcrwo)(enamsr)(rgnslaq)
(owtlasfpdr)(cireaos)(aixecrvuol)(menakspfo)(prgnsla)
(swlac)(cireulm)(alixcrfpjue)(menkasyw)(pnslrqj)
(sotafpgxkj)(celod)(xecrlawit)(menkapd)(prgnsldi)
(sowtlafpvjzc)(cireualov)(lixcrgtq)(menaksput)(rnslu)
(owtsfpyc)(cieuloj)(alixecwtdp)(meakprwsf)(prgslyz)
(otslafpv)(cireulaf)(aixecrldo)(menakspuq)(prgnslaqoc)
(sowlftu)(creiaoq)(alixecrwknd)(menaksocwrv)(pgnsarh)
(swtlafprm)(cieualogkxwb)(liecrwa)(meaksf)(prgsaw)
(sowtfpbejd)(ieualobnj)(alxecrwztdj)(namkspedjuc)(prnslawj)
