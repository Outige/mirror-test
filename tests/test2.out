6
7
4
5
4
5
8
7
5
6
